### ISSUE TITLE
Inconsistent parameter naming Macro __DEFINE_HAS_LVAL_MMBR_FUNC_ 3

### ISSUE THREAD

**Issue by AnantaYudica** [AnantaYudica/basic#11]
```
#include "macro/HasLValueMemberFunction.h"

#define __DEFINE_HAS_LVAL_MMBR_FUNC_TMPL_PARAM_LIST_TPL1(\
    __T_NAME__,\
    __T_RET_NAME__,\
    __T_ARGS_NAME__)\
typename __T_NAME__, typename T1, typename __T_RET_NAME__,\
    typename... __T_ARGS_NAME__

#define __DEFINE_HAS_MMBR_FUNC_PROTO_FUNC_TRUE_TMPL_PARAM_LIST_PFTTPL1(\
    __T_NAME__,\
    __T_RET_NAME__,\
    __T_ARGS_NAME__)\
typename __T_NAME__, typename T1, typename __T_RET_NAME__,\
    typename... __T_ARGS_NAME__

__DEFINE_HAS_LVAL_MMBR_FUNC_(HasLValMmbrFunc4, _HasLValMmbrFunc4, T, Tr, 
    Targs, TPL1, T_NAME_,, PFTTPL1,,, PROTO_FUNC_T_NAME_,,\
    PROTO_FUNC_T_RET_NAME_,, Foo2);
```
parameter macro using name `__PROTO_FUNC_TRUE_TMPL_PARAM_LIST_POSTFIX__` in `__DEFINE_HAS_LVAL_MMBR_FUNC_`
but call back implementation using name `__DEFINE_HAS_MMBR_FUNC_PROTO_FUNC_TRUE_TMPL_PARAM_LIST_`

Similar with #4, #9 and #10

**Comment by AnantaYudica** [comment]
Fixed in v0.1-beta-2

**Comment by AnantaYudica** [comment]
Not a bug, Macro `__DEFINE_HAS_LVAL_MMBR_FUNC_` will be deleted in v0.1-beta-2.
